src/main/java/com/akademia/educators_back/controller/CommentController.java

```diff
@@ -0,0 +1,37 @@
+package com.akademia.educators_back.controller;
+
+import com.akademia.educators_back.service.impl.CommentServiceImpl;
+import com.akademia.educators_back.to.CommentTo;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/comment")
+public class CommentController {
+    private CommentServiceImpl commentServiceImpl;
+
+    public CommentController(CommentServiceImpl commentService) {
```

Comment:
Skoro używamy Lomboka to robimy to w całej apce i w tym wypadku dajemy tylko @AllArgsConstructor